pmg/static/resources/javascript/pmg.js

```diff
@@ -12,6 +12,48 @@ $(function() {
 		});
 	});
 
+
+	var csrftoken = $('meta[name=csrf-token]').attr('content')
+
+	$.ajaxSetup({
+	    beforeSend: function(xhr, settings) {
+	        if (!/^(GET|HEAD|OPTIONS|TRACE)$/i.test(settings.type) && !this.crossDomain) {
+	            xhr.setRequestHeader("X-CSRFToken", csrftoken)
+	        }
+	    }
+	})
+
+	$(".create-alert").on("click", function(e) {
+
+		var q = $(this).data('q'),
+			  committee_id = parseInt($(this).data('committee')) || "",
+				content_type = $(this).data('type');
+
+		$.post(
+			'/user/saved-search/',
+			{
+				q: q,
+				committee_id: committee_id,
+				content_type: content_type
+			}
+		).done(function(resp) {
+			$('.create-alert').addClass('hidden');
+			$('.remove-alert').removeClass('hidden').data('id', resp.id);
+		});
+	});
+
+	$(".remove-alert").on("click", function(e) {
+
+		var id = parseInt($(this).data('id')) || "";
+
+		$.post(
```

Comment:
Ugh, looks like making this a DELETE request is going to be a mission on both sides.

Let's do this instead, send a `POST` to `/user/saved-search/{id}/delete`